networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/Firewall.java

```diff
@@ -58,7 +58,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
       if (cause instanceof WriteTimeoutException) {
         LOG.debug("Firewall closed channel by write timeout. No writes during " + writeTimeout);
       } else {
-        LOG.debug("Error in Firewall, disconnecting" + cause);
+        LOG.debug("Error in Firewall, disconnecting " + cause);
```

Comment:
I'd make this one in classic way: `LOG.debug("Error in Firewall, disconnecting", cause);`

Reply:
Good call, that is better. Thanks!